front/app/containers/Authentication/useSteps/stepConfig/signInFlow.ts

```diff
@@ -88,6 +88,7 @@ export const signInFlow = (
           }
 
           if (requiredCustomFields(requirements.custom_fields)) {
+            // Waarom is dit enkel bij een email sign-in?
```

Comment:
@luucvanderzee Another question: why do we do this step (check if there are custom fields + set current step to "missing-data:custom-fields" for email/password sign-in and not for sign-in via an auth provider?

Reply:
Can you give me more details about the exact situation where you encounter this problem? We already check this after sign-in via SSO- see the `RESUME_FLOW_AFTER_SSO` step in `stepConfig/sharedSteps.ts`.